packages/pools/src/router/types.ts

```diff
@@ -62,7 +62,7 @@ export type SplitTokenInQuote = Quote & {
   /** In amount after fees paid are subtracted. */
   tokenInFeeAmount?: Int;
   swapFee?: Dec;
-  inOutSpotPrice?: Dec;
+  inBaseOutQuoteSpotPrice?: Int;
```

Comment:
Why is this an int now?

Reply:
Since a base amount is now being returned, we've traditionally handled all base amounts as Ints (from SQS, Osmosis node, etc)

Reply:
I'm not sure I follow. Could you elaborate? It's returned as a decimal from SQS:
![image](https://github.com/osmosis-labs/osmosis-frontend/assets/34196718/4611586a-4efc-4177-b78a-cd9e98a86209)